aws/ecs/README.md

```diff
@@ -18,24 +18,25 @@ not remove it and respect the format! -->
 
 | Region         | AMI          |
 |----------------|--------------|
-| us-east-1      | ami-3fc1b928 |
-| us-west-1      | ami-4c2a642c |
-| us-west-2      | ami-de08d4be |
-| eu-west-1      | ami-dba2d8a8 |
-| eu-central-1   | ami-81738eee |
-| ap-northeast-1 | ami-4cbb692d |
-| ap-southeast-1 | ami-bb6cc9d8 |
-| ap-southeast-2 | ami-0de1d16e |
+| us-east-1      | ?? |
```

Comment:
This table is used by the microservice-demo scripts right? Will these "??" be replaced when merged by some script or?